### ISSUE TITLE
"}" symbol breaks the template inside of a string

### ISSUE THREAD

**Issue by lucidd** [neuland/jade4j#101]
If i have a "}" character inside a string in an expression i causes an exception.

```jade
#{"Hello World {}"}
```

```
Caused by: de.neuland.jade4j.exceptions.ExpressionException: unable to evaluate ["Hello World {] - Tokenization de.neuland.jade4j.expression.ExpressionHandler.evaluateExpression@1:15 tokenization error near '..$
 ld { ...'
        at de.neuland.jade4j.expression.ExpressionHandler.evaluateExpression(ExpressionHandler.java:31)
        at de.neuland.jade4j.expression.ExpressionHandler.evaluateStringExpression(ExpressionHandler.java:36)
        at de.neuland.jade4j.compiler.Utils.interpolate(Utils.java:67)
        at de.neuland.jade4j.parser.node.TextNode.execute(TextNode.java:39)
        ... 54 more
Caused by: org.apache.commons.jexl2.JexlException$Tokenization: de.neuland.jade4j.expression.ExpressionHandler.evaluateExpression@1:15 tokenization error near '... ld { ...'
        at org.apache.commons.jexl2.JexlEngine.parse(JexlEngine.java:1258)
        at org.apache.commons.jexl2.JexlEngine.createExpression(JexlEngine.java:435)
        at org.apache.commons.jexl2.JexlEngine.createExpression(JexlEngine.java:419)
        at de.neuland.jade4j.expression.ExpressionHandler.evaluateExpression(ExpressionHandler.java:28)
        ... 57 more
Caused by: org.apache.commons.jexl2.parser.TokenMgrError: Lexical error at line 1, column 15.  Encountered: <EOF> after : "\"Hello World {"
        at org.apache.commons.jexl2.parser.ParserTokenManager.getNextToken(ParserTokenManager.java:3300)
        at org.apache.commons.jexl2.parser.Parser.jj_ntk(Parser.java:3865)
        at org.apache.commons.jexl2.parser.Parser.JexlScript(Parser.java:41)
        at org.apache.commons.jexl2.parser.Parser.parse(Parser.java:24)
        at org.apache.commons.jexl2.JexlEngine.parse(JexlEngine.java:1248)
        ... 60 more
```

If you try the same on the official [jade](http://jade-lang.com/) site it works.

**Comment by chbloemer** [comment]
I added this as a testcase (issues/101.jade) but it's failing also in Version 1.0.0.

**Comment by chbloemer** [comment]
This is fixed in 1.0.7
